cradle-core/src/main/java/com/exactpro/cradle/FetchParameters.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022-2022 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.cradle;
+
+public class FetchParameters {
+    private int fetchSize;
+    private long timeout;
+
+    public FetchParameters()
+    {
+    }
+
+    public int getFetchSize()
+    {
+        return fetchSize;
+    }
+
+    public FetchParameters setFetchSize(int fetchSize)
+    {
+        this.fetchSize = fetchSize;
+        return this;
+    }
+
+    public long getTimeout()
+    {
+        return timeout;
+    }
+
+    public FetchParameters setTimeout(long timeout)
```

Comment:
I thought it could be convenient to write in one string `params = new FetchParameters().setTimeout(100)` and add `setFetchSize()` if it necessary. I will change current behavior and add constructor with parameters.